### ISSUE TITLE
remove CListEventROOT::swapped?

### ISSUE THREAD

**Issue by KrisThielemans** [UCL/STIR#200]
i think this variable is no longer used. In fact, I believe that the [tests on `det1>det2`](https://github.com/UCL/STIR/blob/9ecbf0abd048cd0a8f19250f6cc7b4c199870c26/src/listmode_buildblock/CListRecordROOT.cxx#L91) are not necessary.

@NikEfth , what do you think?

In any case, I propose to wait to fix this (if at all) until PR #189  is merged.

**Comment by NikEfth** [comment]
Actually this code is **buggy**. We have to bring the code from the TOF branch here. I will create an issue and a PR today!

**Comment by KrisThielemans** [comment]
I don't think it is wrong, but would need to check. as long as we have the 2 detectors ok, everything else will work. it could simply be
```C++
  // find det1, det2
 ring1 = _ring1;
 ring2 = _ring2;
```
no need for swapping stuff here.

**Comment by NikEfth** [comment]
I am 60% sure that, when you fixed the TOF aspect, we said this code works for segment 0 (span  1) but not more than that.

**Comment by NikEfth** [comment]
If I recall correctly the segment numbering was not flipped correct under all cases.
